manifest.json

```diff
@@ -115,7 +115,7 @@
 
```

Comment:
Not shown....
`  "custom": {
    "gear-builder": {
      "category": "analysis",
      "image": "flywheel/fsl-topup:0.0.0"
    },
`
should be:

`  "custom": {
    "gear-builder": {
      "category": "analysis",
      "image": "flywheel/fsl-topup:0.0.1"
    },
`